remote-api/src/main/java/org/alfresco/rest/api/impl/RulesImpl.java

```diff
@@ -63,10 +63,12 @@ public class RulesImpl implements Rules
     public CollectionWithPagingInfo<Rule> getRules(final String folderNodeId, final String ruleSetId, final Paging paging)
     {
         final NodeRef folderNodeRef = validateFolderNode(folderNodeId);
-        validateRuleSetNode(ruleSetId, folderNodeRef);
+        final NodeRef ruleSetNodeRef = validateRuleSetNode(ruleSetId, folderNodeRef);
 
+        final boolean isShared = ruleService.isRuleSetShared(ruleSetNodeRef);
         final List<Rule> rules = ruleService.getRules(folderNodeRef).stream()
             .map(Rule::from)
+            .peek(r -> r.setShared(isShared))
```

Comment:
I think we call this every time we call `Rule.from`. Maybe it would make sense to refactor it - we're going to have more methods that return rules as we implement more of the API.